modules/virt-change-vm-instance-type-cli.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vm/virt-creating-vms-from-instance-types.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-change-vm-instance-type-cli_{context}"]
+= Changing the VM instance type by using the CLI
+
+[role="_abstract"]
+As a VM owner, if you want to change the instance type of a VM, you can do this by changing the `name` field in the VM spec. This triggers the update logic, ensuring a new, immutable controller revision snapshot is taken of the new resource configuration.
+
+.Prerequisites
+
+* You created the VM by using an instance type.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Stop the VM.
+
+. Run the following command, replacing `{vm-name}` with the name of your VM, and `{new-instancetype}` with the name of the new instance type you want to change to:
++
+[source,terminal]
+----
+$ oc patch vm/{vm-name} --type merge -p '{"spec":{"instancetype":{"name": "{new-instancetype}"}}'
+----
+
+. Restart the VM.
```

Comment:
This is optional and only required when we can't live update the VM for the user. A user can tell if this is required by looking for the `RestartRequired` condition on the VM after changing the instance type.

Reply:
Good info, thank you!